applications/polkadot-mempool-explorer-v2.md

```diff
@@ -0,0 +1,499 @@
+# polkadot-mempool-explorer-v2
+
+  
+
+  
+
+-  **Team Name:** NA
+
+  
+
+-  **Payment Address:** TBD
```

Comment:
Has this been clarified?

Reply:
@semuelle yes clarified with @Noc2 in private DM at element